src/PeriodEnd/TransferPayments/SFA.DAS.ProviderPayments.Calc.TransferPayments/TransfersProcessor.cs

```diff
@@ -1,32 +1,48 @@
-﻿using MediatR;
+﻿using System.Linq;
+using System.Threading.Tasks;
 using NLog;
 using SFA.DAS.Payments.DCFS.Context;
+using SFA.DAS.ProviderPayments.Calc.TransferPayments.Dependencies;
 
 namespace SFA.DAS.ProviderPayments.Calc.TransferPayments
 {
-    public class TransfersProcessor
+    public partial class TransfersProcessor
     {
         private readonly ILogger _logger;
-        private readonly IMediator _mediator;
+        private readonly IAmATransferRepository _transferRepository;
+        private readonly IProcessLevyTransfers _levyTransferProcessor;
         private readonly ContextWrapper _context;
 
-        public TransfersProcessor(ILogger logger, IMediator mediator, ContextWrapper context)
+        public TransfersProcessor(
+            ILogger logger, 
+            ContextWrapper context, 
```

Comment:
The context isn't actually used and there are other processsors that only use it to get data, and others that don't have it so it can be removed.